src/modules/autocomplete.js

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable object-curly-newline */
+/* eslint-disable object-curly-newline, no-underscore-dangle */
```

Comment:
Can we pull changes to autocomplete / search / recommendations into a separate PR?